test/test_functionalization.py

```diff
@@ -134,6 +165,21 @@ def forward(self, a_1):
     add_tensor = torch.ops.aten.add.Tensor(view_copy_default, fill_scalar);  view_copy_default = fill_scalar = None
     mul_tensor = torch.ops.aten.mul.Tensor(add_tensor, add_tensor);  add_tensor = None
     return mul_tensor
+    """)
+
+        reinplaced_logs = self.get_logs(f, torch.ones(4, 2), reapply_views=True, run_reinplace=True)
+        self.assertExpectedInline(reinplaced_logs, """\
+
+
+
+def forward(self, a_1):
+    empty = torch.ops.aten.empty.memory_format([4, 2], dtype = torch.float32, device = device(type='cpu'), pin_memory = False)
+    fill_scalar = torch.ops.aten.fill_.Scalar(empty, 1.0)
+    view_default = torch.ops.aten.view.default(a_1, [4, 2]);  a_1 = None
+    empty_1 = torch.ops.aten.empty.SymInt([], dtype = torch.float32, device = device(type='cpu'), pin_memory = False)
+    add_tensor = torch.ops.aten.add.Tensor(view_default, empty);  view_default = empty = None
+    mul_tensor = torch.ops.aten.mul_.Tensor(add_tensor, add_tensor)
```

Comment:
Hmm, we probably need to explicitly forbid this. Inplace with aliasing arguments often does not work with eager kernels. Not worth the potential correctness headache.

Reply:
bleh good point. I'll patch this before merging - "if the mutable argument shows up multiple times in the arg list, don't re-inplace"